pegasus_it.ts

```diff
@@ -0,0 +1,548 @@
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="123"/>
+        <source>Left X</source>
+        <translation>Sinistra X</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="129"/>
+        <source>Left Y</source>
+        <translation>Sinistra Y</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="135"/>
+        <source>Left Press</source>
+        <translation>Sinistra Pressione</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="141"/>
+        <source>Right X</source>
+        <translation>Destra X</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="147"/>
+        <source>Right Y</source>
+        <translation>Destra Y</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="153"/>
+        <source>Right Press</source>
+        <translation>Destra Pressione</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="178"/>
+        <source>Settings / Gamepad Layout</source>
+        <translation>Impostazioni / Layout Gamepad</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="182"/>
+        <source>No gamepads connected</source>
+        <translation>Nessun gamepad collegato in questo momento</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="272"/>
+        <source>Hat %1</source>
+        <translation>Hat %1</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="273"/>
+        <source>Button %1</source>
+        <translation>Pulsante %1</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="274"/>
+        <source>Axis %1</source>
+        <translation>Asse %1</translation>
+    </message>
+</context>
+<context>
+    <name>GamepadFooter</name>
+    <message>
+        <location filename="../src/frontend/menu/settings/gamepad/GamepadFooter.qml" line="58"/>
+        <source>Back</source>
+        <translation>Indietro</translation>
+    </message>
+</context>
+<context>
+    <name>GamepadRecorder</name>
+    <message>
+        <location filename="../src/frontend/menu/settings/gamepad/GamepadRecorder.qml" line="107"/>
+        <source>Tilt any axis now to record
+(timeout in %1 seconds)</source>
+        <translation>Sposta qualsiasi delle assi adesso per registrare (timeout in %1 secondi)</translation>

```

Comment:
Please add a line break before the `(timeout in %1 secondi)`, and in the text below this one too.

Reply:
done